android/assets/jsons/translations/Simplified_Chinese.properties

```
@@ -1143,7 +1143,7 @@ You may choose a free Policy = 您可以推行一项免费的政策
 You may choose [amount] free Policies = 您可以推行[amount]项免费的政策
 You gain the [policy] Policy = 你推行了[policy]政策
  # Requires translation!
-You gain the [belief] Belief = 
+You gain the [belief] Belief = 你添加了[belief]信条
```

```suggestion
You gain the [belief] Belief = 你添加了[belief]信仰
```

虽然好像确实“信条”这个翻译更为妥当，但是目前来说unciv大多数地方会belief翻“信仰”，为了一致性建议修改。

（当然还有个选项...干脆ctrl+h一口气信仰全部改信条，然后再commit一个（））